docs/GettingStarted.rst

```diff
@@ -1184,8 +1184,39 @@ The article is shared into the space's activity stream.
 
 
 .. note:: The |image157| button is grey and unclickable until the mandatory fields *title* and *content* are filled. 
+
          
-		  
+Sharing a news from full form
+----------------------------------
+
+To share an article from the full form, you have to display the small form as explained in the previous paragraph and click on "More options" button.
+
+|image159|
+
+The full form contains :
+
+- A **Title** field :  Allows you to enter the news title. The title should not exceed 150 characters: Beyond that limit you will not be allowed to write. This field is mandatory.
```

Comment:
Not sure it is necessary to describe these fields again here, they are already described in the previous chapter. We should only describe the new fields IMO.